api/models.py

```diff
@@ -12,7 +12,6 @@ class UserProfile(models.Model):
 	)
```

Comment:
Please change the double quotes to single quotes, since this is how other two models are using them.